activemq/publisherAMQP.go

```diff
@@ -0,0 +1,74 @@
+package activemq
+
+import (
+	"context"
+	"github.com/andile-innovation/popcorn/log"
+	"net"
+	"pack.ag/amqp"
+	"time"
+)
+
+type NewAMQPPublisherRequest struct {
+	Host string
```

Comment:
Include credentials here:
```suggestion
	Host string
	Username string
	Password string
```